### DISCUSSION TITLE
My profile image is not displaying

### DISCUSSION THREAD

**Discussion by galbitton5** [community/community#64859] [Programming Help]
### Body

I'm using Jekyll with the academicpages.github.io theme for my personal website. Everything seems to be functioning well, but I'm encountering a persistent issue with my profile image not displaying.

Steps I've taken:
- I've uploaded the image profile.png to the root of my repository.
- Updated _config.yml to reference the image: avatar: profile.png.
- I've tried both .jpg and .png formats.
- Tested image link using Markdown on a test page, and the image displayed correctly.
- I've confirmed that GitHub recognizes my email.
However, the image doesn't display in the author profile section.

Repository link: galbitton5.github.io

Any insights or suggestions to resolve this would be greatly appreciated!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arafajs** [comment]
- Make sure that the image file is named `profile.png`. The academicpages theme looks for an image with this exact name in the root of the repository.
- Make sure that the image file is less than `1MB` in size. The academicpages theme has a maximum image size of 1MB.
- Try clearing your browser's cache and cookies. Sometimes this can cause cached images to not display correctly.

**Reply by galbitton5** [reply]
Thank you for your suggestions. I've tried all the following:

- Made sure the image size is under 1MB.
- Cleared my browser's cache and cookies.
- Checked the image file format and name.
Despite these attempts, the profile image still doesn't display as expected, and instead, I see a default symbol of a person.

Do you have any other ideas or insights that might help resolve this issue?

**Reply by arafajs** [reply]
- Double check that the image file name matches exactly what is specified in `_config.yml` for the avatar parameter.
- Make sure that the image file is accessible to the web server. You can do this by trying to access the image file directly in your browser. If you can't access the image file, then the web server may not be able to find it.
- Try using a different image file. Sometimes a particular image file may not work with the academicpages theme for some reason. Try using a different image file and see if that works.

If you've tried all of these things and the image still isn't displaying, then it's possible that there is a bug in the academicpages theme.

**Comment by Jaber-Saed** [comment]
I appreciate your reaching out regarding the issue you're facing with your Jekyll-based personal website using the academicpages.github.io theme. It's unfortunate that your profile image isn't displaying correctly, despite having followed the appropriate steps.

You've taken the initiative to upload the profile.png image to the root of your repository and have updated the _config.yml file to reference the image as avatar: profile.png. You've also tested the image link using Markdown on a test page, and it displayed correctly.

Given these steps, it's somewhat puzzling why the image isn't showing up in the author profile section. Let's explore a few additional possibilities that might help resolve this issue:

File Name and Location: Although you've tried different formats (.jpg and .png), ensure that the file name and its case match exactly between the file on the repository and its reference in _config.yml. Also, confirm that the image is indeed in the root directory.

GitHub Pages Build: After making changes, GitHub Pages needs to rebuild your site. Sometimes, there might be a delay between updates and the actual display. Give it a little time and then refresh the website to see if the image appears.

Browser Cache: Cached data in your browser can sometimes cause images not to load. Try opening your website in a private browsing window or clearing your browser cache and then reloading the page.

Image URL Format: Double-check the URL in your _config.yml file. Make sure it follows the correct format, including the proper file extension. For instance, if your website URL is https://username.github.io, the avatar URL should be https://username.github.io/profile.png.

Markdown Syntax: Confirm that you're using the correct Markdown syntax to display the image. It should resemble something like ![Profile Image](/profile.png).

GitHub Pages Configuration: Verify your GitHub Pages settings in your repository's settings. Sometimes, configuration issues can impact the rendering of elements on your site.

In case you've tried all these steps and the issue persists, it might be beneficial to consult the theme's documentation or its GitHub repository. Theme-specific considerations could play a role in this situation.

I'm hopeful that one of these solutions will help you overcome the challenge and successfully display your profile image on your website. If you encounter any further obstacles, don't hesitate to seek more assistance. Your website is a reflection of your identity, and it's important to have it accurately represent you. Best of luck!

**Reply by galbitton5** [reply]
I greatly appreciate all the insightful suggestions!
A friend and I found a workaround by using an external URL. We uploaded the image to "postimage", copied the URL into the _config.yml file, and it resolved the issue.
I hope this conversation can assist others facing similar challenges in the future.

Many thanks for your dedication and help!
